domains/Qwertx.is-an.app.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
The filename should be `qwertx.is-an.app.json`

Reply:
Ok